vortex-serde/src/layouts/read/footer.rs

```diff
@@ -73,8 +72,24 @@ impl Footer {
             .read_layout(footer_bytes, loc, scan, message_cache)
     }
 
+    pub fn dtype_bytes(&self) -> VortexResult<Bytes> {
+        let start_offset = self.leftovers_schema_offset();
+        let end_offset = self.leftovers_layout_offset();
+        let bytes = self
+            .leftovers
+            .slice(start_offset + FLATBUFFER_SIZE_LENGTH..end_offset);
+        // Run validation on dtype bytes
+        self.fb_schema()?;
+        Ok(bytes)
```

Comment:
why does this return `Bytes` instead of `fb::Schema`, since with Bytes we just end up having to validate the fb twice?

Reply:
I need an owned type, you can see that subsequent access aren't validated

Reply:
plus this logic is literally almost identical to `self.fb_schema()`, aside from return type munging